node/sync/locators/src/block_locators.rs

```diff
@@ -248,10 +263,12 @@ impl<N: Network> BlockLocators<N> {
         }
```

Comment:
In line 253, could we write `*current_height > 0` instead of `*current_height != last_height`? The error message makes it clear that the assumption is that `last_height` is 0 at this point, which is indeed the case.